### PR TITLE
Routing

### LINKED ISSUE #1: Add page structure
- [x] Install React Router
- [x] Add React Router
- [x] Add Routes

### PR DESCRIPTION
**Pull request by lauravikanis**
- Add React Router + Routes close #1 
- Component adjustments

-> I added Routes to my project. Doing so showed me a lot of adjustments I had to do in my components.

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -5,30 +5,38 @@ import tourPath from "../assets/tour.svg";
...
-        <h2>FAVORITEN</h2>
-        <img src={favoritePath} alt="favoriteicon" />
-      </Card>
+      <Link to="/choice">
+        <Card primary>
+          <h2>TOUR</h2>
+          <img src={tourPath} alt="touricon" />
+        </Card>
+      </Link>
+      <Link to="search">
```
Comment (miriamhaenle) [inline client/src/pages/Home.js:28]: I think you missed a / here
```suggestion
      <Link to="/search">
```

File: client/src/pages/Location.js [hunk 2]
```diff
@@ -30,12 +36,24 @@ const Tour = () => {
         </p>
       </Card>
       <ImageCard>
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
-        <img src={placeholder} alt="placeholder" />
+        <Link to="/Pictures">
```
Comment (miriamhaenle) [inline client/src/pages/Location.js:39]: usually urls are written with small letters
```suggestion
        <Link to="/pictures">
```

File: client/src/pages/Pictures.js [hunk 3]
```diff
@@ -10,22 +10,22 @@ const PicturesDiv = styled.div`
...
+      <PageHeadline>Bildname</PageHeadline>
       <PictureContainer>
         <img src={placeholder} alt="placeholder" />
       </PictureContainer>
-      <PageHeadline>Suche</PageHeadline>
       <p>
-        Der Dom. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
-        diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
-        erat, sed diam voluptua.{" "}
+        Bildbeschreibung Lorem ipsum dolor sit amet, consetetur sadipscing
```
Comment (miriamhaenle) [inline client/src/pages/Pictures.js:24]: Don't forget to remove the `Lorem ipsum` text in the end :)

File: client/src/pages/Tour.js [hunk 4]
```diff
@@ -21,12 +23,18 @@ const Tour = () => {
         incidunt.
       </p>
       <Standort>
-        <li>Standort</li>
-        <li>Standort</li>
-        <li>Standort</li>
-        <li>Standort</li>
-        <li>Standort</li>
-        <li>Standort</li>
+        <Link to="/Location">
```
Comment (miriamhaenle) [inline client/src/pages/Tour.js:26]:
```suggestion
        <Link to="/location">
```

File: client/src/components/Card/Card.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ const Card = styled.div`
   background: ${(props) => (props.primary ? "var(--primary-color)" : "white")};
   border-radius: 15px;
   width: 100%;
-  max-width: 500px;
+  /* max-width: 500px; */
```
Comment (miriamhaenle) [inline client/src/components/Card/Card.js:11]: No commented out code on main :) You can always check back in your git history if you want to reuse previous code

**Review by miriamhaenle — COMMENTED** [review]
Nice job 👍 I noticed that you mix English and German language in the app (all the links in the router are English but some other text is German). Something you can look into in a future branch.

File: client/src/pages/Pictures.js (hunk 3 above)
Comment (lauravikanis) [reply client/src/pages/Pictures.js:24]: At the time its just a placeholder as a want to read the data from a database wich is not yet set up :)

**Comment by lauravikanis** [comment]
> Nice job 👍 I noticed that you mix English and German language in the app (all the links in the router are English but some other text is German). Something you can look into in a future branch.

Thanks for your feedback. I'll adjust my code with all the reviews. You were writing about the German-English-confusement. I'll probably adjust this in the next branches as most of them are still placeholders. :)

**Pull request merged**